index.html

```diff
@@ -3,10 +3,144 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
-  <title>Document</title>
+  <link rel="stylesheet" href="/css/Nabvar.css">
+  <link rel="stylesheet" href="/css/burgerMenuWindows.css">
+  <link rel="stylesheet" href="/css/dinamicSection.css">
+  <title>CapstoneFernando</title>
 </head>
 <body>
-  
+  <div class="upVar">
+    <ul class="Uplist">
+      <a href=""><img src="/images/facebook.png" class="logFaceAndTwit" srcset=""></a>
+      <a href=""><img src="/images/twitter.png" class="logFaceAndTwit" srcset=""></a>
+      <a href="">English</a>
+      <a href="">My page</a>
+      <a href="">Logout</a>
+    </ul>
+  </div>
+  <header class="header">
+    <div>
+      <nav class="mobile-navbar">
+        <button><img src="./images/mobile_menu 1.png" alt="burgerMenu" onclick ="opneNave();" class="image-burger-menu"></button>
+      </nav>
+    </div>
+    
+    <div class="container-logo">
+      <a href="index.html"><img src="./images/LogoUrban.png" alt= My Logo class="logo"></a>
+    </div>
+      <nav class="navbar">
+        <ul class="nav-list"> 
+          <li>
+            <a href="AboutPage.html">About</a>
+          </li>
+          <li>
+            <a href="#Program">Program</a>
+          </li>
+          <li>
+            <a href="#Join">Join</a>
+          </li>
+          <li>
+          <li>
+            <a href="#Sponsor">Sponsor</a>
+          </li>
+          <li>
+            <a href="#News">News</a>
+          </li>
+            <a href="#" class="header_button" id="header_button">UC Campaigns</a>
+          </li>
+        </ul>
+      </header>
+
+  <main>
+    <section class="mobile-menu" id="mobileNave">
+      <ul class="nav-list2"> 
+        <img onclick="closeNave();" class="img-close" src="./images/Icon_X_.png" alt="">
+        <li class="mobile-link" onclick="closeNave();"><a href="AboutPage.html" >About</a></li>
+        <li class="mobile-link" onclick="closeNave();"><a href="#aboutme-container" >Program</a></li>
+        <li class="mobile-link" onclick="closeNave();"><a href="#contact">News</a></li>
+    </section>
+
+    <section class="home-page" id="home-page" >
+      <div class="containerHome-Page">
+        <div>
+          <p class="greetingOne">"Hello, Sharing World!"</p>
+          <h1 class="home-title" id="home-title" >Urban Art Festival Creativity in the Streets 2023</h1>
+        </div>
+        <div class="home-Description">
+          <p>
+            "Join us at the exhilarating 'Creativity in the Streets' Urban Art Festival, where the most talented street artists come together to transform the streets of our city into an outdoor art gallery. This annual festival celebrates urban culture, artistic expression, and the creative community in a vibrant and unique atmosphere."
+          </p>
+        </div>
+        <div>
+          <p class="home-date">2023.07.03(THU) ~ 12(WEN)</p>
```

Comment:
- [ ] If you could take a moment to review your main page on mobile, you might notice that the alignment of the paragraph and main title isn't quite matching the Behance design. 

   ### Your design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/9639ebde-43bf-4e47-b18d-9af9d3c06452)

   ### Behance design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/176e9227-cd68-49f0-8922-aa26892d31e8)